orte/mca/schizo/ompi/schizo_ompi.c

```diff
@@ -979,6 +980,10 @@ static int setup_fork(orte_job_t *jdata,
         opal_setenv("OMPI_MCA_orte_bound_at_launch", "1", true, &app->env);
     }
 
+    char cache_line_size[16];
+    snprintf(cache_line_size, 16, "%d", opal_cache_line_size);
+    opal_setenv("OMPI_MCA_opal_cache_line_size", cache_line_size, true, &app->env);
```

Comment:
If you use `OPAL_MCA_PREFIX` to read the env variable, you should use it to set the variable, too.